.github/workflows/test.yml

```diff
@@ -9,8 +9,7 @@ on:
 env:
   FOUNDRY_PROFILE: lite
   MAINNET_RPC_URL: ${{ secrets.MAINNET_RPC_URL }}
-  ARBITRUM_RPC_URL: ${{ secrets.ARBITRUM_RPC_URL }}
-  OPTIMISM_RPC_URL: ${{ secrets.OPTIMISM_RPC_URL }}
+  BASE_RPC_URL: ${{ secrets.BASE_RPC_URL }}
```

Comment:
optional ditto comment as other PR: whether it's possible to just use same api key for both MAINNET_RPC_URL and BASE_RPC_URL by default

Reply:
Same as the other PR on uni-v3-lib, keeping them separate so we can use different node providers for different chains.